lib/App/DuckPAN/Cmd/New.pm

```diff
@@ -226,7 +233,39 @@ sub run {
 		$self->app->emit_and_exit(-1, $error)
 	}
 
-	$self->app->emit_info("Success!");
+	$self->app->emit_info('Success!');
+}
+
+# Allow user to choose a handler
+sub _config_handler {
+	my $self = shift;
+
+	my @handlers = (
+		'remainder: (default) The query without the trigger words, spacing and case are preserved.',
+		'query_raw: Like remainder but with trigger words intact',
+		'query: Full query normalized with a single space between terms',
+		'query_lc: Like query but in lowercase',
+		'query_clean: Like query_lc but with non-alphanumeric characters removed',
+		'query_nowhitespace: All whitespace removed',
+		'query_nowhitespace_nodash: All whitespace and hyphens removed',
+		'words: Like query_clean but returns an array of the terms split on whitespace',
+		'query_parts: Like query but returns an array of the terms split on whitespace',
+		'query_raw_parts: Like query_parts but array contains original whitespace elements'
```

Comment:
should we add "matches"? We'd need to indicate it only works with a `regex` triggers though...